### PR TITLE
Create "Linked site page changes" diagram and explanation

### PR DESCRIPTION
**Pull request by tzhelyazkova**
Task: [T271949](https://phabricator.wikimedia.org/T271949)

### CODE REVIEW

File: systems/WikibaseClient/05-Building_Block_View.md [hunk 1]
```diff
@@ -21,6 +21,16 @@
 
 ## Linked Site Page Changes
 
+Responsible for notifying a repository that a page that is linked to by an entity has changed.
```
Comment (addshore) [inline systems/WikibaseClient/05-Building_Block_View.md:24]: We should perhaps sync this line of description up with that which is currently in the table above.
I wonder if it should also only be in one of the 2 places.

**Review by addshore — CHANGES_REQUESTED** [review]
While looking at this diagram I can't help but think again that it could be nice to be able to show the external system that is being interacted with on the diagram.
I was thinking this while making the change related building block diagrams.

I think that would bring it closer to https://docs.arc42.org/section-5/
If we think thats the right direction then I would also go and change the other diagrams.

**Comment by tzhelyazkova** [comment]
> While looking at this diagram I can't help but think again that it could be nice to be able to show the external system that is being interacted with on the diagram.
> I was thinking this while making the change related building block diagrams.
> 
> I think that would bring it closer to https://docs.arc42.org/section-5/
> If we think thats the right direction then I would also go and change the other diagrams.

Do you mean, we should have `WikibaseRepo` drawn as an external system for all those block diagrams? Wouldn't that be the same thing already shown in the Whitebox Overall diagram on the same page?

**Comment by addshore** [comment]
> > While looking at this diagram I can't help but think again that it could be nice to be able to show the external system that is being interacted with on the diagram.
> > I was thinking this while making the change related building block diagrams.
> > I think that would bring it closer to https://docs.arc42.org/section-5/
> > If we think thats the right direction then I would also go and change the other diagrams.
> 
> Do you mean, we should have `WikibaseRepo` drawn as an external system for all those block diagrams? Wouldn't that be the same thing already shown in the Whitebox Overall diagram on the same page?

In this situation yes perhaps, but in other situations the connections to external systems end up highlighting important things, and I feel we need to be consistent in the diagrams.

**Review by addshore — APPROVED** [review]
Looks good, I'm going to make a PR bringing the other diagram in this md file that I made more inline with the one added here as I prefer it.

**Comment by addshore** [comment]
That PR is https://github.com/wmde/wikidata-wikibase-architecture/pull/35

**Pull request merged**